src/content/docs/faq.md

```diff
@@ -451,3 +451,48 @@ If you want to know more, we recommend reading
 [this excellent article by @orhun](https://blog.orhun.dev/stdout-vs-stderr/).
 
 [`BufWriter`]: https://doc.rust-lang.org/std/io/struct.BufWriter.html
+
+## How do I avoid panics due to out of range calls on the Buffer?
+
+In general, most of the code in tui-rs and hence Ratatui was not designed around using `Result`s to
```

Comment:
Maybe just say `Ratatui` without mentioning tui-rs? I think it may come across unintentionally as us blaming tui-rs for legacy design decisions.

Reply:
Yep - good idea.